CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,55 @@
+# Changelog 1.12
+
+## 1.12.0 - 2023-05-24
+
+- The Multicast, TopologyAwareHints, NodeIPAM features are graduated from Alpha to Beta and therefore enabled by default.
+
+### Added
+
+- Add a new field 'logLabel' to ACNP and ANP CRDs to specify firewall log label for Audit Logging. ([#4748](https://github.com/antrea-io/antrea/pull/4748), [@qiyueyao])
+- Add support for short-circuiting in AntreaProxy to ensure that the traffic from Pod/Node clients to
+external addresses behaves the same way as the traffic from external clients to external addresses. ([#4815](https://github.com/antrea-io/antrea/pull/4815), [@hongliangl])
+- Add support for ExternalIP in AntreaProxy to allows a Service to be accessed from outside the cluster using a static IP address. ([#4866](https://github.com/antrea-io/antrea/pull/4866), [@hongliangl])
+- Add two new fields 'sourcePort' and 'sourceEndPort' in Antrea-native policy API to match traffic initiated from specific ports. ([#4687](https://github.com/antrea-io/antrea/pull/4687), [@Dyanngg])
+- Add OVS table name as label for 'ovs_flow_count' Prometheus metrics. ([#4893](https://github.com/antrea-io/antrea/pull/4893), [@cr7258])
+- Add WireGuard tunnel mode for Multi-cluster to establish encrypted connection between member clusters. ([#4606](https://github.com/antrea-io/antrea/pull/4606), [@hjiajing])
+- Watch EndpointSlice API by default for exported Service's endpoint change in Multi-cluster. ([#4895](https://github.com/antrea-io/antrea/pull/4895), [@luolanzone])
+
+### Changed
+
+- Deprecate Antrea Octant Plugin. ([#4825](https://github.com/antrea-io/antrea/pull/4825), [@antoninbas])
```

Comment:
Question - we do not have any new UI enhancements in this release?

Reply:
UI is in another repo, I suppose there are some changes. Should we add UI PRs into this release log?
@antoninbas any suggestion for the UI change?

Reply:
There is no need to mention UI changes as they are tracked separately.
However, you could mention the UI here:

> Deprecate Antrea Octant Plugin; it is replaced by a dedicated Antrea UI.

And also it could be mentioned in the release announcement for Antrea v1.12